### PR TITLE
feat(mojaloop/#3533): mojaloop helm release v15.2.0

### PR DESCRIPTION
**Pull request by oderayi**
feat(mojaloop/#3533): mojaloop helm release v15.2.0 - https://github.com/mojaloop/project/issues/3533
- Upgraded packages to their latest versions as follows:
  - ml-api-adapter: [v14.0.0 --> v14.0.4](https://github.com/mojaloop/ml-api-adapter/compare/v14.0.0...v14.0.4)
  - central-ledger: [v17.0.3 --> v17.3.0](https://github.com/mojaloop/central-ledger/compare/v17.0.3...v17.3.0) 
  - account-lookup-service: [v14.1.0 --> v14.2.5](https://github.com/mojaloop/account-lookup-service/compare/v14.1.0...v14.2.5)
  - quoting-service: [v15.0.2 --> v15.5.0](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.5.0)
  - central-settlement: [v15.0.0 --> v15.0.4](https://github.com/mojaloop/central-settlement/compare/v15.0.0...v15.0.4) 
  - bulk-api-adapter: [v15.0.3 --> v16.0.1](https://github.com/mojaloop/bulk-api-adapter/compare/v15.0.3...v16.0.1)
  - email-notifier: [v12.0.0 --> v13.0.1](https://github.com/mojaloop/email-notifier/compare/v12.0.0...v13.0.1)
  - transaction-requests-service: [v14.1.0 --> v14.1.1](https://github.com/mojaloop/transaction-requests-service/compare/v14.1.0...v14.1.1)
  - event-sidecar: [v12.0.0 --> v13.0.1](https://github.com/mojaloop/event-sidecar/compare/v12.0.0...v13.0.1)
  - mojaloop-simulator: [v14.0.1 --> v15.0.0](https://github.com/mojaloop/mojaloop-simulator/compare/v14.0.1...v15.0.0)
  - sdk-scheme-adapter: [v23.0.1 --> v23.1.1](https://github.com/mojaloop/sdk-scheme-adapter/compare/v23.0.1...v23.1.1)
  - ml-testing-toolkit: [v16.1.1 --> v17.0.0](https://github.com/mojaloop/ml-testing-toolkit/compare/v16.1.1...v17.0.0)
  - auth-service: [v14.0.0 --> v14.0.1](https://github.com/mojaloop/auth-service/compare/v14.0.0...v14.0.1)
  - als-consent-oracle: [v0.2.0 --> v0.2.1](https://github.com/mojaloop/als-consent-oracle/compare/v0.2.0...v0.2.1)
  - thirdparty-api-svc: [v13.0.2 --> v13.0.4](https://github.com/mojaloop/thirdparty-api-svc/compare/v13.0.2...v13.0.4)
  - thirdparty-sdk: [v15.1.0 --> v15.1.1](https://github.com/mojaloop/thirdparty-sdk/compare/v15.1.0...v15.1.1)

See [v15.2.0 release note](https://github.com/mojaloop/helm/blob/master/.changelog/release-v15.2.0.md) for more details about this release.

### CODE REVIEW

File: .changelog/release-v15.2.0.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+
+- For *BREAKING CHANGES*, please review the section `#5` "Breaking Changes" below.
+- For *KNOWN ISSUES*, please review the section `#8` "Known Issues" below.
+
+## 0. Summary
+
+Minor enhancements for [v15.1.0 Release](https://github.com/mojaloop/helm/blob/master/.changelog/release-v15.1.0.md), which includes the following changes:
+
+
+1. Characterisation that improves performance and metrics around some core services: ALS, Central Ledger, Quoting Service.
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:16]:
```suggestion
1. Performance & instrumentation improvements for the following core services: Account-Lookup-Service, Quoting-Service, ML-API-Adapter & Central Ledger.
```

File: .changelog/release-v15.2.0.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+2. NodeJS Upgrades
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:17]:
```suggestion
2. Maintenance upgrades for all packaged services, this includes NodeJS & dependencies.
```

File: .changelog/release-v15.2.0.md [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+3. Bugfixes
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:18]:
```suggestion
3. Several bug-fixes mainly due to the maintenance upgrades, and a result of the Performance Characterisation work to improve stability of Mojaloop core services under load
```

File: .changelog/release-v15.2.0.md [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+3. account-lookup-service: 14.1.0 -> [v14.2.5](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.5) ([Compare](https://github.com/mojaloop/account-lookup-service/compare/v14.1.0...v14.2.5))
+4. quoting-service: v15.0.2 -> [v15.4.0](https://github.com/mojaloop/quoting-service/releases/tag/v15.4.0) ([Compare](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.4.0))
+5. central-settlement: v15.0.0 -> [v15.0.4](https://github.com/mojaloop/central-settlement/releases/tag/v15.0.4) ([Compare](https://github.com/mojaloop/central-settlement/compare/v15.0.0...v15.0.4))
+6. bulk-api-adapter: v15.0.3 -> [v16.0.1](https://github.com/mojaloop/bulk-api-adapter/releases/tag/v16.0.1) ([Compare](https://github.com/mojaloop/bulk-api-adapter/compare/v15.0.3...v16.0.1))
+7. central-event-processor: [v12.0.0](https://github.com/mojaloop/central-event-processor/releases/tag/v12.0.0) *(Refer to section 5. BREAKING CHANGES)*
+8. email-notifier: [v12.0.0](https://github.com/mojaloop/email-notifier/releases/tag/v12.0.0) *(Refer to section 5. BREAKING CHANGES)*
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:68]: Note that we do have a new version of this service that was upgraded. However we have no way of testing it :(

File: .changelog/release-v15.2.0.md [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+9. als-oracle-pathfinder: [v12.0.0](https://github.com/mojaloop/als-oracle-pathfinder/releases/tag/v12.0.0)
+10. transaction-requests-service: v14.0.1 -> [v14.1.1](https://github.com/mojaloop/transaction-requests-service/releases/tag/v14.1.1) ([Compare](https://github.com/mojaloop/transaction-requests-service/compare/v14.1.0...v14.1.1))
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:70]:
```suggestion
10. transaction-requests-service: v14.1.0 -> [v14.1.1](https://github.com/mojaloop/transaction-requests-service/releases/tag/v14.1.1) ([Compare](https://github.com/mojaloop/transaction-requests-service/compare/v14.1.0...v14.1.1))
```

File: .changelog/release-v15.2.0.md [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+14. operator-settlement: [v11.0.0](https://github.com/mojaloop/operator-settlement/releases/tag/v11.0.0) *(DEPRECATED)*
+15. event-sidecar: v12.0.0 -> [v13.0.1](https://github.com/mojaloop/event-sidecar/releases/tag/v13.0.1) ([Compare](https://github.com/mojaloop/event-sidecar/compare/v12.0.0...v13.0.1))
+16. event-stream-processor: [v12.0.0-snapshot.7](https://github.com/mojaloop/event-stream-processor/releases/v12.0.0-snapshot.7)
+17. simulator: [12.0.0](https://github.com/mojaloop/simulator/releases/tag/v12.0.0)
+18. mojaloop-simulator: v14.0.1 -> [v15.0.0](https://github.com/mojaloop/mojaloop-simulator/releases/tag/v15.0.0) ([Compare](https://github.com/mojaloop/event-sidecar/compare/v14.0.1...v15.0.0))
+19. sdk-scheme-adapter: v23.0.1 -> [v23.1.1](https://github.com/mojaloop/sdk-scheme-adapter/releases/tag/v23.1.1) ([Compare](https://github.com/mojaloop/sdk-scheme-adapter/compare/v23.0.1...v23.1.1))
+20. ml-testing-toolkit: v16.1.1 -> v17.0.0(https://github.com/mojaloop/ml-testing-toolkit/releases/tag/v17.0.0) ([Compare](https://github.com/mojaloop/sml-testing-toolkit/compare/v16.1.1...v17.0.0))
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:80]:
```suggestion
20. ml-testing-toolkit: v16.1.1 -> [v17.0.0](https://github.com/mojaloop/ml-testing-toolkit/releases/tag/v17.0.0) ([Compare](https://github.com/mojaloop/sml-testing-toolkit/compare/v16.1.1...v17.0.0))
```

File: .changelog/release-v15.2.0.md [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+21. ml-testing-toolkit-ui: v15.3.0 -> [v15.4.0](https://github.com/mojaloop/ml-testing-toolkit-ui/releases/tag/v15.3.0) ([Compare](https://github.com/mojaloop/ml-testing-toolkit-ui/compare/v15.3.0...v15.4.0))
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:81]: Looks like no change here...

```suggestion
21. ml-testing-toolkit-ui: [v15.4.0](https://github.com/mojaloop/ml-testing-toolkit-ui/releases/tag/v15.3.0)
```

File: .changelog/release-v15.2.0.md [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+22. ml-testing-toolkit-client-lib: [v1.2.0](https://github.com/mojaloop/ml-testing-toolkit-client-lib/releases/tag/v1.2.0)
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:82]: Was this upgrade in the Helm chart?

```suggestion
22. ml-testing-toolkit-client-lib: v1.2.0 -> [v1.2.1](https://github.com/mojaloop/ml-testing-toolkit-client-lib/releases/tag/v1.2.1)  ([Compare](https://github.com/mojaloop/ml-testing-toolkit-ui/compare/v1.2.0...v1.2.1))
```

File: .changelog/release-v15.2.0.md [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+23. auth-service: v14.0.0 -> [v14.0.1](https://github.com/mojaloop/auth-service/releases/tag/v14.0.1) ([Compare](https://github.com/mojaloop/auth-service/compare/v14.0.0...v14.0.1))
+24. als-consent-oracle: v0.2.0 -> [v0.2.0](https://github.com/mojaloop/als-consent-oracle/releases/tag/v0.2.1) ([Compare](https://github.com/mojaloop/als-consent-oracle/compare/v0.2.0...v0.2.1))
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:84]:
```suggestion
24. als-consent-oracle: v0.2.0 -> [v0.2.1](https://github.com/mojaloop/als-consent-oracle/releases/tag/v0.2.1) ([Compare](https://github.com/mojaloop/als-consent-oracle/compare/v0.2.0...v0.2.1))
```

**Review by mdebarros — COMMENTED** [review]
Some comments/suggestions for the release notes.

File: .changelog/release-v15.2.0.md (hunk 8 above)
Comment (aaronreynoza) [reply .changelog/release-v15.2.0.md]: Yes, the helm chart was updated accordingly, but image was missing the bump. Just pushed it.

File: bulk-centralledger/Chart.yaml [hunk 10]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Bulk Services Helm chart for Kubernetes
 name: bulk-centralledger
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline bulk-centralledger/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: bulk-centralledger/chart-handler-bulk-transfer-fulfil/Chart.yaml [hunk 11]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Bulk Transfer Fulfil Handler Helm chart for Kubernetes
 name: cl-handler-bulk-transfer-fulfil
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline bulk-centralledger/chart-handler-bulk-transfer-fulfil/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: bulk-centralledger/chart-handler-bulk-transfer-get/Chart.yaml [hunk 12]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Bulk Transfer Get Handler Helm chart for Kubernetes
 name: cl-handler-bulk-transfer-get
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline bulk-centralledger/chart-handler-bulk-transfer-get/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: bulk-centralledger/chart-handler-bulk-transfer-processing/Chart.yaml [hunk 13]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Bulk Transfer Processing Handler Helm chart for Kubernetes
 name: cl-handler-bulk-transfer-processing
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline bulk-centralledger/chart-handler-bulk-transfer-processing/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-handler-admin-transfer/Chart.yaml [hunk 14]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Transfer Prepare Handler Helm chart for Kubernetes
 name: centralledger-handler-admin-transfer
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.2.0
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-admin-transfer/Chart.yaml:4]: why 14.2.0 directly and not v14.1.0?

File: centralledger/chart-handler-admin-transfer/Chart.yaml [hunk 15]
```diff
@@ -1,8 +1,8 @@
...
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-admin-transfer/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/Chart.yaml [hunk 16]
```diff
@@ -14,33 +14,35 @@ maintainers:
...
     repository: "file://./chart-handler-transfer-fulfil"
     condition: centralledger-handler-transfer-fulfil.enabled
   - name: centralledger-handler-timeout
-    version: 14.0.0
+    version: 14.1.0
     repository: "file://./chart-handler-timeout"
     condition: centralledger-handler-timeout.enabled
   - name: centralledger-handler-admin-transfer
-    version: 14.0.0
+    version: 14.2.0
```
Comment (elnyry-sam-k) [inline centralledger/Chart.yaml:45]: may have to change based on centralledger/chart-handler-admin-transfer/Chart.yaml comment

File: centralledger/chart-handler-timeout/Chart.yaml [hunk 17]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Timeout Handler Helm chart for Kubernetes
 name: centralledger-handler-timeout
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-timeout/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-handler-transfer-fulfil/Chart.yaml [hunk 18]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Transfer Fulfil Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-fulfil
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-transfer-fulfil/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-handler-transfer-get/Chart.yaml [hunk 19]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Get Transfer Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-get
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-transfer-get/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-handler-transfer-position/Chart.yaml [hunk 20]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Transfer Position Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-position
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-transfer-position/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-handler-transfer-prepare/Chart.yaml [hunk 21]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Transfer Prepare Handler Helm chart for Kubernetes
 name: centralledger-handler-transfer-prepare
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-handler-transfer-prepare/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: centralledger/chart-service/Chart.yaml [hunk 22]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Central-Ledger Service Helm chart for Kubernetes
 name: centralledger-service
-version: 14.0.0
-appVersion: "17.0.3"
+version: 14.1.0
+appVersion: "17.2.1"
```
Comment (elnyry-sam-k) [inline centralledger/chart-service/Chart.yaml:5]:
```suggestion
appVersion: "17.3.0"
```

File: mojaloop-bulk/Chart.yaml [hunk 23]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Mojaloop Bulk Helm chart for Kubernetes
 name: mojaloop-bulk
-version: 15.0.0
-appVersion: "bulk-api-adapter: v15.0.3; central-ledger: v17.0.3"
+version: 16.0.0
+appVersion: "bulk-api-adapter: v16.0.0; central-ledger: v17.3.0"
```
Comment (elnyry-sam-k) [inline mojaloop-bulk/Chart.yaml:5]:
```suggestion
appVersion: "bulk-api-adapter: v16.0.1; central-ledger: v17.3.0"
```

File: centralledger/Chart.yaml (hunk 16 above)
Comment (oderayi) [reply centralledger/Chart.yaml:45]: @elnyry-sam-k Minor version bump here seem okay since we bumped the minor version in the appVersion.

Comment (elnyry-sam-k) [reply centralledger/Chart.yaml:45]: minor version is fine but wanted to know if there's a reason to skip v14.1.0 and go from x.0.0 to x.2.0

File: .changelog/release-v15.2.0.md [hunk 24]
```diff
@@ -0,0 +1,294 @@
...
+2. [#2322 - Helm install failing with with "medium to large" release names](https://github.com/mojaloop/project/issues/2322)
+3. [#2317 - Mojaloop Helm deployments are not compatible when deployed to ARM-arch based hosts](https://github.com/mojaloop/project/issues/2317)
+4. [#2892 - Disabled DFSP showing getParty info](https://github.com/mojaloop/project/issues/2892)
+5. [#2435 - Quoting-Service is incorrectly handling failed responses to FSPs when forwarding requests](https://github.com/mojaloop/project/issues/2435)
+6. Test issues causing instability/intermitant failures on Test Case Results
+    1. [#2717 - Thirdparty TTK Test-Collection is not repeatable](https://github.com/mojaloop/project/issues/2717)
+    2. [#2845 - QA: Replace Legacy-Simulator as a NORESPONSE_SIMPAYEE in Testing-Toolkit Goden Path Test-Suite](https://github.com/mojaloop/project/issues/2845)
+    3. [#3027 - QA: Mojaloop Helm v14.1.0 Release - Bulk Tests fail on first run](https://github.com/mojaloop/project/issues/3027)
+    4. [#2925 - Helm Test Intermittent failure with 'Generic ID not found](https://github.com/mojaloop/project/issues/2925)
```
[the remark this answers is not in the document]
Comment (oderayi) [reply .changelog/release-v15.2.0.md:151]: @elnyry-sam-k This issue is still open.

File: mojaloop/Chart.yaml [hunk 25]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 description: Mojaloop Helm chart for Kubernetes
 name: mojaloop
-version: 15.1.0
-appVersion: "ml-api-adapter: v14.0.0; central-ledger: v17.0.3; account-lookup-service: v14.1.0; quoting-service: v15.0.2; central-settlement: v15.0.0; bulk-api-adapter: v15.0.3; transaction-requests-service: v14.1.0; simulator: v12.0.0; mojaloop-simulator: v14.0.1; sdk-scheme-adapter: v23.0.1; auth-service: v14.0.0; consent-oracle: v0.2.0; thirdparty-sdk: v13.0.2; ml-testing-toolkit: v16.1.1; ml-testing-toolkit-ui: v15.4.0;"
+version: 15.2.0
+appVersion: "ml-api-adapter: v14.0.4; central-ledger: v17.3.0; account-lookup-service: v14.2.5; quoting-service: v15.2.2; central-settlement: v15.0.3; bulk-api-adapter: v16.0.0; transaction-requests-service: v14.1.1; simulator: v12.0.0; mojaloop-simulator: v15.0.0; sdk-scheme-adapter: v23.1.1; auth-service: v14.0.1; consent-oracle: v0.2.1; thirdparty-sdk: v15.1.1; ml-testing-toolkit: v17.0.0; ml-testing-toolkit-ui: v15.4.0;"
```
Comment (mdebarros) [inline mojaloop/Chart.yaml:5]: @oderayi take note that the `ml-testing-toolkit-ui` has not been bumped?

Comment (oderayi) [reply mojaloop/Chart.yaml:5]: Yes @mdebarros, there was no update to ttk-ui in this release

File: .changelog/release-v15.2.0.md (hunk 24 above)
[the remark this answers is not in the document]
Comment (elnyry-sam-k) [reply .changelog/release-v15.2.0.md]: yes it is.. was wondering if we saw this happen in recent times.. but lets leave it for now, thanks

**Review by mdebarros — DISMISSED** [review]
+1

**Review by mdebarros — APPROVED** [review]
+1

**Review by mdebarros — APPROVED** [review]
+1 once other comments have been addressed.

File: .changelog/release-v15.2.0.md [hunk 26]
```diff
@@ -0,0 +1,157 @@
...
+
+Enhancements and non-breaking changes to the [v15.1.0 Release](https://github.com/mojaloop/helm/blob/master/.changelog/release-v15.1.0.md), which includes:
+
+1. Nodejs version of core and supporting services upgraded to v18.17.1 LTS
+2. Performance improvements and metrics around core services: ALS, Central Ledger, Quoting Service based on characterisation and fixes, enhancements done
+3. General maintenance and bug fixes
+
+## 1. New Features
+
+1. **Performance fixes, enhancements and maintenance:**
```
Comment (mdebarros) [inline .changelog/release-v15.2.0.md:21]: I just noticed that some of these updates were missed/swallowed: 

- mojaloop/#3519: add topic name override config map logic (https://github.com/mojaloop/central-ledger/issues/969) ([b99d091](https://github.com/mojaloop/central-ledger/commit/b99d09129f328aab60e4059035be4d263ecd9f98)), closes https://github.com/mojaloop/project/issues/3519 - https://github.com/mojaloop/central-ledger/releases/tag/v17.2.0
- mojaloop/#3498: alter message keys for prepare, fulfil and timeout (https://github.com/mojaloop/central-ledger/issues/965) ([db3e3b2](https://github.com/mojaloop/central-ledger/commit/db3e3b2f0450f1093f1f20eb4f49b4f4de53ddd0)), closes https://github.com/mojaloop/project/issues/3498 - from https://github.com/mojaloop/central-ledger/releases/tag/v17.1.0
- json stringify on every response (https://github.com/mojaloop/central-ledger/issues/961) ([6798bfd](https://github.com/mojaloop/central-ledger/commit/6798bfda51be18c2d44c22e10ef67bc76c8b2195)) - https://github.com/mojaloop/central-ledger/releases/tag/v17.0.5
- mojaloop/#2880: simplify liquidity cover check and fix issue with validation (for main) (https://github.com/mojaloop/central-ledger/issues/918) ([825faf7](https://github.com/mojaloop/central-ledger/commit/825faf7c1c918b12eddedeb97eea0a2d563457e4)) - https://github.com/mojaloop/central-ledger/releases/tag/v17.0.4

Its possible that we have missed others. I recommend double checking this list against the issue report for the PI to double double check.

**Review by mdebarros — CHANGES_REQUESTED** [review]

File: .changelog/release-v15.2.0.md [hunk 27]
```diff
@@ -0,0 +1,157 @@
...
+    - the Quoting service operating mode to be set quoting-service.config.simple_routing_mode_enabled=true (in helm mojaloop/values.yaml under quoting-service config). If this is incorrectly configured, it will result in several failures in the quoting-service tests (7 expected failures). If this is disabled, ensure that you update the corresponding test-case environment variable parameter **SIMPLE_ROUTING_MODE_ENABLED** ( in helm mojaloop/values.yaml ml-testing-toolkit -> extraEnvironments.hub-k8s-default-environment.json.inputValues) to match.
+    - the **on-us transfers** (in mojaloop/values.yaml "enable_on_us_transfers: false" under centralledger-handler-transfer-prepare -> config and  cl-handler-bulk-transfer-prepare -> config) configuration to be disabled. The test-case environment variable parameter (**ON_US_TRANSFERS_ENABLED** (in helm mojaloop/values.yaml ml-testing-toolkit -> extraEnvironments.hub-k8s-default-environment.json.inputValues), the same name used on postman collections) must similarly match this value.
+
+4. Simulators
```
Comment (elnyry-sam-k) [inline .changelog/release-v15.2.0.md:117]:
```suggestion
## 4. Simulators
```

File: .changelog/release-v15.2.0.md [hunk 28]
```diff
@@ -0,0 +1,157 @@
...
+
+1. [#2119 - Idempotency for duplicate quote request](https://github.com/mojaloop/project/issues/2119)
+2. [#2322 - Helm install failing with with "medium to large" release names](https://github.com/mojaloop/project/issues/2322)
+3. [#2317 - Mojaloop Helm deployments are not compatible when deployed to ARM-arch based hosts](https://github.com/mojaloop/project/issues/2317)
+4. [#2435 - Quoting-Service is incorrectly handling failed responses to FSPs when forwarding requests](https://github.com/mojaloop/project/issues/2435)
+5. Test issues causing instability/intermitant failures on Test Case Results
+    1. [#2717 - Thirdparty TTK Test-Collection is not repeatable](https://github.com/mojaloop/project/issues/2717)
+    2. [#2925 - Helm Test Intermittent failure with 'Generic ID not found](https://github.com/mojaloop/project/issues/2925)
+
+## 9. Contributors
```
Comment (elnyry-sam-k) [inline .changelog/release-v15.2.0.md:150]:
```suggestion
## 7. Contributors
```

**Review by mdebarros — DISMISSED** [review]
Null

File: .changelog/release-v15.2.0.md (hunk 26 above)
Comment (oderayi) [reply .changelog/release-v15.2.0.md:21]: @aaronreynoza is looking into this and will update.

**Review by mdebarros — DISMISSED** [review]
+1

**Comment by elnyry-sam-k** [comment]
Looks good so far, thanks @oderayi .. we can move forward with a newer version of the quoting service, when it is ready.

File: .changelog/release-v15.2.0.md [hunk 29]
```diff
@@ -82,7 +82,7 @@ Enhancements and non-breaking changes to the [v15.1.0 Release](https://github.co
 1. ml-api-adapter: v14.0.0 -> [v14.0.4](https://github.com/mojaloop/ml-api-adapter/releases/tag/v14.0.4) ([Compare](https://github.com/mojaloop/ml-api-adapter/compare/v14.0.0...v14.0.4))
 2. central-ledger: v17.0.3 -> [v17.3.0]((https://github.com/mojaloop/central-ledger/releases/tag/v17.3.0)) ([Compare](https://github.com/mojaloop/central-ledger/compare/v17.0.3...v17.3.0))
 3. account-lookup-service: 14.1.0 -> [v14.2.5](https://github.com/mojaloop/account-lookup-service/releases/tag/v14.2.5) ([Compare](https://github.com/mojaloop/account-lookup-service/compare/v14.1.0...v14.2.5))
-4. quoting-service: v15.0.2 -> [v15.4.0](https://github.com/mojaloop/quoting-service/releases/tag/v15.4.0) ([Compare](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.4.0))
+4. quoting-service: v15.0.2 -> [v15.5.0](https://github.com/mojaloop/quoting-service/releases/tag/v15.4.0) ([Compare](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.5.0))
```
Comment (elnyry-sam-k) [inline .changelog/release-v15.2.0.md:85]:
```suggestion
4. quoting-service: v15.0.2 -> [v15.5.0](https://github.com/mojaloop/quoting-service/releases/tag/v15.5.0) ([Compare](https://github.com/mojaloop/quoting-service/compare/v15.0.2...v15.5.0))
```

**Review by elnyry-sam-k — DISMISSED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**